pkg/task/restore.go

```diff
@@ -86,9 +88,17 @@ func (cfg *RestoreConfig) ParseFromFlags(flags *pflag.FlagSet) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
 	if cfg.Config.Concurrency == 0 {
 		cfg.Config.Concurrency = defaultRestoreConcurrency
 	}
+	if cfg.Config.Concurrency < defaultRestoreConcurrency {
+		fmt.Fprintf(os.Stdout,
```

Comment:
why print to stdout?

Reply:
I didn't print it to log, because I think not all user will check log after restore, and when user set unsuitable config to start command, I think we should let them know immediately

Reply:
Stdout is completely hidden when using SQL though.